src/app/modules/group/pages/group-details/group-details.component.ts

```diff
@@ -15,6 +15,7 @@ export class GroupDetailsComponent {
   state$ = this.groupDataSource.state$.pipe(mapStateData(state => ({
     group: withManagementAdditions(state.group),
     route: state.route,
+    breadcrumbs: state.breadcrumbs,
```

Comment:
If the breadcrumb is not used in groups-details, it should not be in the state.

Reply:
I was replicating what was done with item, but it's probably not a good idea
However removing it from the state is not satisfying either because the `GroupData` interface from `group-datasource` is used by group components and coupling their `groupData` type to a page state type is also a bad idea.
I see 2 viable solutions:
1. Enhance the group with management additions, and spread the rest of the state as is to avoid breaking the type instead of specifying each property one by one
2. Make subcomponents having a `groupData` input  retrieve the group data value from the data source and not as an input anymore

Reply:
Although I think option 2 is more appropriate, it's not done this way in other parts of the app. So I opted for option 1.
Fixed

Reply:
Yes, as I wrote somewhere else: Option 2 is only possible because the datasource was thought to bypass a technical issue of passing arg to sub-components built by routing. But now that is not required anymore, DS should be private to group-by-id. So opt2 should not be used as it is a way to use a global variable for storing the group (considering there is only 1 group in the app at a given moment).